marshmallow_enum/__init__.py

```diff
@@ -38,7 +38,7 @@ def __init__(
         self.enum = enum
         self.by_value = by_value
 
-        if error and any(old in error for old in ('{name', '{value', '{choices')):
+        if error and any(old in error for old in ('name}', 'value}', 'choices}')):
```

Comment:
I changed it because as it was before I would always get the warning. Even when using the not deprecated interpolations. But, I just noticed that the pytest version should be >= 3.8.0, otherwise I'm not getting any warnings. Could you confirm this?

Reply:
I'll check it out. The issue with checking for the `}` at the end is there could be formatting directives in between the the target and the end of the interpolation.